src/test/java/com/epam/tc/hw5/pages/UserTablePage.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.tc.hw5.pages;
+
+import com.epam.tc.hw5.pages.sections.LogArea;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class UserTablePage extends BasePage {
+
+    public LogArea logArea;
+
+    public UserTablePage(WebDriver webDriver) {
+        super(webDriver);
+        this.logArea = new LogArea(webDriver);
+        this.url = "https://jdi-testing.github.io/jdi-light/user-table.html";
+    }
+
+    //Table columns
+    @FindBy(css = "#user-table>tbody>tr>td:nth-child(2)>select")
+    public List<WebElement> numberTypeSelectors;
+    @FindBy(css = "#user-table>tbody>tr>td:nth-child(3)>a")
+    public List<WebElement> users;
+    @FindBy(css = "#user-table>tbody>tr>td:nth-child(4)>div.user-descr>span")
+    public List<WebElement> descriptionTexts;
+    @FindBy(css = "#user-table>tbody>tr>td:nth-child(4)>div.user-descr>input[type='checkbox']")
+    public List<WebElement> checkBoxes;
+
+    public List<WebElement> getColumn(String columnName) {
+        if (columnName.equalsIgnoreCase("Number Type Dropdowns")) {
+            return numberTypeSelectors;
+        } else if (columnName.equalsIgnoreCase("Usernames")) {
+            return users;
+        } else if (columnName.equalsIgnoreCase("Description texts under images")) {
+            return descriptionTexts;
+        } else if (columnName.equalsIgnoreCase("checkboxes")) {
+            return checkBoxes;
+        } else {
+            throw new NoSuchElementException("Failed to find column " + columnName);
+        }
+    }
+
+    public WebElement getRowItem(int rowNumber, String itemType) {
+        String rowCssSelector = "#user-table>tbody>tr:nth-child(" + rowNumber + ")";
```

Comment:
it's better to use templates in such cases, like "#user-table>tbody>tr:nth-child%s"

Reply:
spent much time on this part, but couldn't use correct template with pass test.